source/aks/redeploying-aks-clusters.html.md.erb

```diff
@@ -1,95 +1,93 @@
+- If not updated, you can change manually providing PR has been approved and merged [View Jenkins Configuration](https://build.platform.hmcts.net/configure)
 
-* Once the pipeline to deploy the cluster has completed, refer to the snapshot previously taken and create a new disk from it named as **jenkins-disk** ensuring it is placed in the location referenced [here](https://github.com/hmcts/cnp-flux-config/blob/85d61449e8633c6a975798c01e7ce155c9861c7e/apps/jenkins/jenkins/sbox-intsvc/disk.yaml#L8).
+## Upgrading the Cluster
 
-#### After deployment of a cluster
+Find the cluster you want to upgrade into the portal > Cluster Configuration > Upgrade Version > in the Kubernetes version drop down menu choose the latest GA version > Click Save
 
-Once the cluster has been redeployed and Jenkins is back up and running, test the [cnp-plum-recipes-service](https://sandbox-build.platform.hmcts.net/job/HMCTS_Sandbox_CNP/job/cnp-plum-recipes-service/job/master/) job again to confirm it runs successfully as it did before (it should).
+This will trigger the upgrade of the cluster, which will take some time. Once this has finished you need to create a PR in the [SDS repo](https://github.com/hmcts/aks-sds-deploy/blob/master/environments/aks/ptlsbox.tfvars#L3) / [CFT repo](https://github.com/hmcts/aks-cft-deploy/blob/main/environments/aks/demo.tfvars#L3) changing the kubernetes_version value
+  * Note: The kubernetes_version value only takes major version values, i.e 1.25, 1.26 and so on.
 
-### ITHC
+Once the changes have been reviewed, approved and merged, run the [SDS Pipeline](https://dev.azure.com/hmcts/PlatformOperations/_build?definitionId=482) / [CFT Pipeline](https://dev.azure.com/hmcts/PlatformOperations/_build?definitionId=766). Once completed compare the hrs and pods statuses with the ones you saved before the upgrade took place.
 
-#### Before deployment of a cluster
-- Remove the cluster you are going to redeploying from the AGW. [PR example here](https://github.com/hmcts/azure-platform-terraform/pull/546)
-- Unsure which IP belongs to which cluster? Check the front end IP of the kubernetes-internal loadbalancer
+## After upgrading of a cluster
+- Add the cluster back into AGW once you have confirmed the upgrade has been successful. [PR example for SDS here](https://github.com/hmcts/sds-azure-platform/pull/474/files) / [PR example for CFT here](https://github.com/hmcts/azure-platform-terraform/pull/595/files)
 
-#### After deployment of a cluster
-- Add the cluster back into AGW once you have confirmed deployment has been successful. [PR example here](https://github.com/hmcts/azure-platform-terraform/pull/560)
+## Re-deploying a cluster
 
+<%= warning_text('Re-building a cluster is only done when absolutely necessary.') %>
```

Comment:
it seems a bit weird to go into environment specifics without saying how to do a rebuild.
I would suggest moving this section to here:
http://127.0.0.1:4567/aks/redeploying-aks-clusters.html#to-re-create-a-cluster

and then say read on for any environment specifics